src/backends/backend.js

```diff
@@ -26,7 +26,24 @@ class LocalStorageAuthStore {
 
 const slugFormatter = (template = "{{slug}}", entryData) => {
   const date = new Date();
-  const identifier = entryData.get("title", entryData.get("path"));
+
+  const getIdentifier = (entryData) => {
+	  const validIdentifierFields = ["title", "path"];
+	  const identifiers = validIdentifierFields.map((field) => {
+	  	return entryData.find((_, key) => {
+	  		return key.toLowerCase() === field;
+	  	});
+	  });
+	  
+	  const identifier = identifiers.find(i => typeof i !== 'undefined');
```

Comment:
Please don't use the single-letter variable `i` to mean anything except a loop index. `ident` might be a good replacement here. `typeof i !== 'undefined'` can also be replaced with either `i === undefined` (exactly equivalent to what you have here, and more readable) or `_.isNil` from lodash (will also catch `null` values) - the `typeof` is unnecessary.